.gitignore

```diff
@@ -1,4 +1,5 @@
 .env
 .env.*
 !.env.example
+.tool-versions
```

Comment:
Hah, actually I do use asdf! I figured I'd keep this out of the repo so not everyone gets affixed to one particular Node version though — I always find it kind of annoying when I go to a repo and am forced to install a new version because there's a `.tool-versions` in there. Does that make sense in this instance? The project should work on a variety of Node versions shouldn't it?